requirements.txt

```diff
@@ -1,4 +1,4 @@
-koji
+koji < 1.20
```

Comment:
I don't think we should do this:

- this prevents installing alt-src together with a koji >= 1.20, so you can't easily test it against a newer koji to see if the problem has been fixed.  So *if* we need to constrain to a particular version due to some bug (not intentional API breakage), I think test-requirements.txt is the better place to do it - that can fix CI without preventing you from installing whatever versions you want for manual testing.
- it'll break / require changes to python-alt-src RPM (see how the .spec file handles requirements.txt)
- this is just postponing handling the issue (and is there anything filed to follow up on this?)
- looking into the actual problem, it seems to be that koji 1.20 decided to make rpm-py-installer optional, so we're not getting it installed any more in our CI; but if you try to use koji features without having it, you just get these crashes.

So I think the correct solution here is not to put any range on koji, but rather to add rpm-py-installer into test-requirements.txt.

Reply:
- I wasn't sure how to go about handling this requirement. So added a constraint initially on koji to have a working code. However, removed that constraint and moved the dependency to test-requirements as suggested.
- python-alt-src doesn't include koji from requirements.txt during the build. Also, the koji rpm available is 1.19 but that doesn't eliminate the risk
- filed an issue for this https://github.com/release-engineering/alt-src/issues/22

Reply:
> python-alt-src doesn't include koji from requirements.txt during the build

I was wrong on this point - I saw that it was patched via sed and I thought that the way it was patched would break if the requirements.txt line included a version range.  However I see now that's not the case, it would filter out the entire line correctly.